src/dbnode/storage/index/options.go

```diff
@@ -33,6 +34,14 @@ import (
 const (
 	// defaultIndexInsertMode sets the default indexing mode to synchronous.
 	defaultIndexInsertMode = InsertSync
+
+	// documentArrayPool size in general: 256*256*sizeof(doc.Document)
+	// = 256 * 256 * 16
+	// = 1mb (but with Go's heap probably 2mb)
+	// TODO(r): Make this configurable in a followup change.
```

Comment:
We're probably gonna want this tunable really soon so we don't have to ask people to recompile when they're testing our changes, especially if their read workloads are very different from ours

Reply:
This is now only going to be used by the compactor, so it should be tiny (we never will have 256 concurrent compactions, and compactor just uses a single pool).